test/functional/util/ImportUtils.ts

```diff
@@ -41,7 +38,7 @@ describe("ImportUtils.importOrRequireFile", () => {
         expect(exports.default).to.be.a("function")
         expect(exports.number).to.be.eq(6)
 
-        rmdirSync(testDir)
+        await fs.rmdir(testDir, { recursive: true })
```

Comment:
Indentation

Reply:
Indentation is correct. The file was formatted before commit.